templates/infinispan-ephemeral.json

```diff
@@ -102,7 +186,7 @@
         "name": "${APPLICATION_NAME}"
       },
       "spec": {
-        "replicas": 3,
+        "replicas": "${NUMBER_OF_INSTANCES}",
```

Comment:
you need to use the double bracket syntax when doing non-string parameter values:
https://docs.openshift.org/latest/dev_guide/templates.html#writing-parameters